docs/configuration/cosmos-conf.rst

```diff
@@ -54,6 +54,23 @@ This page lists all available Airflow configurations that affect ``astronomer-co
     - Default: ``None``
     - Environment Variable: ``AIRFLOW__COSMOS__DBT_DOCS_CONN_ID``
 
+.. _enable_profile_cache:
```

Comment:
So we are consistent with naming (e.g. #1014), let's use: `enable_cache_profile`. This way, all configs related to enabling cache share the same prefix.

Also, let's set this by default - so we are consistent with pre-existing cache.